server/controllers/userController.js

```diff
@@ -92,10 +92,10 @@ class UserController {
     }, 'RETURNING id');
 
     const { id } = fetchedUser;
-    const tokenObj = { id };
+    if (!isadmin) { tokenObj = token({ id }); }
 
```

Comment:
This `if` should be broken into new lines after the opening curly brackets. Let all if statements look alike, since others are broken in seperate lines.